keda/assets/logs/keda.yaml

```
@@ -0,0 +1,61 @@
+id: keda
+metric_id: keda
+backend_only: false
+installation_sources:
+  - keda
+  - keda-admission-webhooks
+  - keda-metrics-apiserver
+facets:
+pipeline:
+  type: pipeline
+  name: Keda
+  enabled: true
+  filter:
+    query: source:(keda OR keda-admission-webhooks OR keda-metrics-apiserver)
+  processors:
+    - type: grok-parser
+      name:  Keda Parser
+      enabled: true
+      source: message
+      samples:
+        - 'I1216 14:53:06.261024       1 main.go:112] "msg"="Connecting Metrics Service gRPC client to the server" "address"="keda-operator.keda.svc.cluster.local:9666" "logger"="keda_metrics_adapter"'
+        - 'I1216 14:53:06.680302       1 tlsconfig.go:243] "Starting DynamicServingCertificateController"'
+        - '2024-12-23T14:21:51Z	ERROR	scale_handler	error resolving auth params	{"type": "ScaledJob", "namespace": "default", "name": "csv-processor", "triggerIndex": 0, "error": "connection to redis failed: dial tcp 10.244.0.6:6379: connect: no route to host"}'
+        - '2024-12-23T14:21:38Z	INFO	KubeAPIWarningLogger	unknown field "status.authenticationsTypes"'
+        - '2024-12-16T14:53:12Z	INFO	controller-runtime.builder	Registering a validating webhook	{"GVK": "keda.sh/v1alpha1, Kind=ScaledObject", "path": "/validate-keda-sh-v1alpha1-scaledobject"}'
+      grok:
+        matchRules: |
+          keda_metrics_adapter3 %{_level_adapter}%{_date_adapter}\s+%{_pid}.*\s+%{_logger_adapter}]\s+"msg"=%{doubleQuotedString:msg} %{data::keyvalue}
+          keda_metrics_adapter2 %{_level_adapter}%{_date_adapter}\s+%{_pid}.*\s+%{_logger_adapter}]\s+%{_msg}
+          keda_metrics_adapter1 %{_level_adapter}%{_date_adapter}\s+%{_pid}.*\s+%{_logger_adapter}]\s+"%{data::keyvalue}" %{data:component.name:keyvalue}
+          keda_operator2 %{_date_operator}\s+%{_level_operator}\s+%{_logger_op}\s+%{_msg}\t%{data::json}
```

```suggestion
          keda_operator2 %{_date_operator}\s+%{_level_operator}\s+%{_logger_op}\s+%%{regex("[^\\t]*"):msg}\t%{data::json}
```

Following the same logic as before, we should try to get rid of unneeded usage of `data` when we have a separator to break on.
Tested it, and it should work as well.